nucypher/cli/commands/workers.py

```diff
@@ -0,0 +1,122 @@
+"""
```

Comment:
This module is cloud specific (at least for now), and `worker` is fairly entrenched in our parlance. Should we call rename this module to something that uses the term `cloud`? eg. ` `cloudworkers` or `clouddeployment` (a little wordy) or ...? 

We use `Worker`/`Ursula` interchangeably so that way we are running the cli as:

`nucypher cloudworkers` instead of `nucypher workers`.